source/fundamentals/connection/connect.txt

```diff
@@ -87,11 +87,25 @@ Other Connection Targets
 Connect to Atlas
 ~~~~~~~~~~~~~~~~
 
-In the following example, the driver uses a sample connection URI to
-connect to a MongoDB instance on Atlas, using placeholders for your
-username and password. In this example, the client uses
-the :ref:`Stable API <csharp-stable-api>` feature, though
-you can connect to Atlas without this feature.
+To connect to an Atlas deployment, you need to create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the ``MongoClient``
+constructor. To specify your connection URI, pass it to the ``FromConnectionString()``
+method, creating a ``MongoClientSettings`` instance. To specify any other
+settings, set the relevant fields of the settings object.
```

Comment:
Suggestion:
I think it would be easier to read if the terminology was consistent (client options vs settings). I think in this case, it's more confusing to make the switch in terminology because the class name is MongoClientSettings.

```suggestion
client options, set the relevant fields of the MongoClientSettings object.
```

Reply:
changed all references to options to settings